### DISCUSSION TITLE
How to use my homelab server when use dnf enable copr command

### DISCUSSION THREAD

**Discussion by BH1SCW** [fedora-copr/copr#2957] [Q&A] [answered]
Hi all,

I try to setup my local copr service using docker, everything works well now and after I built a package
How can I add my local repo using command

```
sudo dnf copr enable testuser/hello-world
```

the defaut settings will try to add:

copr.fedorainfracloud.org/testuser/hello-world

I am not sure if I need to patch dnf?

Thanks

**Comment by FrostyX** [comment, marked as answer]
There is a feature in the DNF Copr plugin called "hubs",  please take a look at `man dnf-copr`:

```
--hub Copr
      Specify a Copr hub to use. Default is the Fedora Copr: https://copr.fedorainfracloud.org.
```

You need to create a configuration for your hub in `/etc/dnf/plugins/copr.d/`:

```
Configuration file should contain a section for each hub, each section having hostname
      (mandatory), protocol (default https) and port (default 443) parameters.:

         [fedora]
         hostname = copr.fedorainfracloud.org
         protocol = https
         port = 443
```

That way you will be able to use both Fedora Copr and your Copr at the same time. 

If you only want to use yours, the easiest way would be going to the [API page](https://copr.fedorainfracloud.org/api) of your Copr instance and downloading the `~/.config/copr` config.

**Reply by BH1SCW** [reply]
Thanks, I am grateful for your help~
